packages/maps/src/components/MapView.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from "react";
+import NativeMapView from "./ReactNativeMaps";
+import { MapViewProps } from "@draftbit/types";
+
+// Dynamically import from ./ReactNativeMaps so that we don't
```

Comment:
is this a TODO or is this what's happening?

Reply:
Yeah, that's what's happening